src/app/admin/volunteers/volunteers-details/volunteers-details.component.ts

```diff
@@ -21,15 +23,11 @@ export class VolunteersDetailsComponent implements OnInit, OnDestroy {
     private route: ActivatedRoute,
     private volunteerFacade: VolunteersFacade,
     private translateService: TranslateService,
-  ) {}
-
-  ngOnInit() {
-    this.volunteer$
-      .pipe(
-        filter((volunteer) => !!volunteer),
-        takeUntil(this._destroy),
-      )
-      .subscribe();
+  ) {
+    this.route.data.pipe(take(1)).subscribe((data) => {
```

Comment:
There is a use case where `take(1)` won't do the job.
For example if we navigate from one volunteer to another volunteer. The URL will change, but the data will be from previous volunteer.

My suggestion would be to replace `take(1)` with `takeUntil(this._destroy)`